README.md

```diff
@@ -919,20 +919,40 @@ The same example with flags:
 However, keep in mind that Storers have to support regular expression and depending on the implementation of the storage handler the accepted syntax may vary.
 An error of `ErrNotImplemented` will be returned for those storage backends which do not support the `$regex` operator.
 
+Having an `schema.Array` field with values being `schema.Object`:
```

Comment:
Should probably rewrite this a bit.

> The `$elemMatch` operator matches documents that contain an array field with at least one element that matches all the specified query criteria.

No mention that the element needs to be an object.

Reply:
Done.